src/main/java/org/fundacion/jala/converter/models/User.java

```diff
@@ -59,104 +59,117 @@ public User(final String newName, final String newPassword, final String newToke
         this.id = newId;
     }
 
     /**
-     * Obtains Name of the User
-     * @return name of the user
+     * Obtains Name of the User.
+     *
+     * @return name of the user.
      */
     public String getName() {
         return name;
     }
 
     /**
-     * Sets new name of the user
-     * @param newName String with the new name
+     * Sets new name of the user.
+     *
+     * @param newName is a String with the new name.
      */
     public void setName(final String newName) {
         this.name = newName;
     }
 
     /**
-     * Obtains password of the User
-     * @return password of the user
+     * Obtains password of the User.
+     *
+     * @return password of the user.
      */
     public String getPassword() {
         return password;
     }
 
     /**
-     * Sets new password of the User
-     * @param newPassword String with the password
+     * Sets new password of the User.
+     *
+     * @param newPassword is a String with the password.
      */
     public void setPassword(final String newPassword) {
         this.password = newPassword;
     }
 
     /**
-     * Obtains token of the user
-     * @return code token of the user
+     * Obtains token of the user.
+     *
+     * @return String with code token of the user.
      */
     public String getToken() {
         return token;
     }
 
     /**
-     * Sets new token of the User
-     * @param newToken String with the token
+     * Sets new token of the User.
+     *
+     * @param newToken is a String with the token.
      */
     public void setToken(final String newToken) {
         this.token = newToken;
     }
 
     /**
-     * Obtains all the projects that realized the user
-     * @return all the projects of the user
+     * Obtains all the projects that realized the user.
+     *
+     * @return all the Assets of the user.
      */
     public List<Asset> getAssets() {
         return assets;
     }
 
     /**
-     * Sets projects of the user
-     * @param newProjects a list of projects
+     * Sets projects of the user.
+     *
+     * @param newProjects a list of projects.
      */
     public void setAssets(final List<Asset> newProjects) {
         this.assets = newProjects;
     }
 
     /**
-     * Obtains all the projects that realized the user
-     * @return all the projects of the user
+     * Obtains all the projects that realized the user.
+     *
+     * @return all the projects of the user.
      */
     public List<Project> getProjects() {
         return projects;
     }
 
     /**
-     * Sets projects of the user
-     * @param newProjects a list of projects
+     * Sets projects of the user.
+     *
+     * @param newProjects is a list of projects.
      */
     public void setProjects(final List<Project> newProjects) {
         this.projects = projects;
     }
 
     /**
-     * Obtains all date of the user
-     * @return String of the dates
+     * Obtains all date of the user.
```

Comment:
I believe it should be "data" instead of "date". On the other hand, the method seems to transforms the user object to a String.

Reply:
ok

it was solved